src/stk/molecular/topology_graphs/topology_graph/optimizer/utilities.py

```diff
@@ -0,0 +1,92 @@
+"""
+This module defines utilities for optimizers.
+
+"""
+
+from mchammer import Bond
+
+
+class OptimizationIncompleteError(Exception):
+    ...
+
+
+def get_mch_bond_topology(state):
+    """
+    Returns bonds with atom1_id < atom2_id and long bonds to optimize.
+
+    """
+
+    long_bond_ids = []
+    mch_bonds = []
+    for i, bond_infos in enumerate(state.get_bond_infos()):
+        ba1 = bond_infos.get_bond().get_atom1().get_id()
+        ba2 = bond_infos.get_bond().get_atom2().get_id()
+        # Must ensure bond atom id ordering is such that
+        # atom1_id < atom2_id.
+        if ba1 < ba2:
+            mch_bonds.append(
+                Bond(id=i, atom1_id=ba1, atom2_id=ba2)
```

Comment:
Make MCHammer do the ordering, not the user of MCHammer

Reply:
Done.